test/clients.go

```diff
@@ -38,26 +39,30 @@ type Clients struct {
 // and register with the Context
 type CleanupFunc func() error
 
-// Setup creates the context object needed in the e2e tests
-func Setup(t *testing.T) *Context {
-	clients, err := NewClients()
-	if err != nil {
-		t.Fatalf("Couldn't initialize clients: %v", err)
-	}
-
-	ctx := &Context{
-		T:       t,
-		Clients: clients,
+// Setup creates context objects for all kubeconfigs passed from the command line
+func Setup(t *testing.T) []*Context {
```

Comment:
I wonder if this should always just return the Admin context and if we should create `SetupView` and `SetupEdit` or something that specifically craetes the other two contexts.

That way, we can also add a "name" string to the context to be used to print enough info in tests without providing that additionally.

Reply:
OK. Probably makes sense because we already depend on certain order of parameters to `--kubeconfig`, so we will always know that the second kubeconfig is for `edit` role and third for `view` role